libobs/obs-internal.h

```diff
@@ -414,11 +425,13 @@ struct obs_core_data {
 	pthread_mutex_t services_mutex;
 	pthread_mutex_t audio_sources_mutex;
 	pthread_mutex_t draw_callbacks_mutex;
+	pthread_mutex_t canvases_mutex;
 	DARRAY(struct draw_callback) draw_callbacks;
 	DARRAY(struct rendered_callback) rendered_callbacks;
 	DARRAY(struct tick_callback) tick_callbacks;
 
-	struct obs_view main_view;
+	/* Main canvas, guaranteed to exist for the lifetime of the program. */
```

Comment:
This is something of a sentence fragment, so you can probably drop the period, or change it to use complete sentences.